spec/system/page_objects/pages/user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class User < PageObjects::Pages::Base
+      def find(selector)
```

Comment:
Don't we want to return self for these functions? If not what would be the rule?

Reply:
Actually I have some ideas.. mention you in the other PR. https://github.com/discourse/discourse-topic-voting/pull/132/files